### ISSUE TITLE
Can this drive a JVC bubble tube A68ADT25X01?

### ISSUE THREAD

**Issue by chuckhacker** [tdaede/td-crt#5]
JVC’s curved bubble tube, the A68ADT25X01, was used for 10 years throughout their consumer TV lines.

The tube was used so long by JVC because of its great design. Even TVs from 20 years ago have no geometry issues and very minor focus issues. The quality is superb. At this point, it is easier to come across the tube than an actual D-Series TV, as prices continue to inflate.

So if it's possible to come up with an open source driver board for this CRT tube, it would certainly fill a niche for enthusiasts like myself.

**Comment by tdaede** [comment]
Yes, it should be able to. That tube (and the equivalent Toshiba models used for MS8/MS9 etc) should work well. The circuit is designed to use the original yoke, so if the yoke is bonded it should still be OK.

Depending on the particular inductance of the yoke, higher scan rates (24-31kHz) can be more difficult, though if it's not more than the ~2mH that most of these are it should be OK.
